Sources/PhotoCaptureViewController/PhotoCaptureViewController.swift

```diff
@@ -386,52 +382,23 @@ open class PhotoCaptureViewController: UIViewController, PhotoCollectionViewLayo
             return
         }
 
-        guard let controller = imagePickerAdapter?.viewControllerForImageSelection({ assets in
-            if let waitView = self.imagePickerWaitingForImageDataView, assets.count > 0 {
-                waitView.translatesAutoresizingMaskIntoConstraints = false
-                self.view.addSubview(waitView)
-
-                waitView.removeConstraints(waitView.constraints.filter({ (constraint: NSLayoutConstraint) -> Bool in
-                    constraint.secondItem as? UIView == self.view
-                }))
-                self.view.addConstraint(NSLayoutConstraint(item: waitView, attribute: .centerX, relatedBy: .equal, toItem: self.view, attribute: .centerX, multiplier: 1, constant: 0))
-                self.view.addConstraint(NSLayoutConstraint(item: waitView, attribute: .centerY, relatedBy: .equal, toItem: self.view, attribute: .centerY, multiplier: 1, constant: 0))
-
-                switch UIDevice.current.orientation {
-                case .landscapeRight:
-                    self.view.addConstraint(NSLayoutConstraint(item: waitView, attribute: .height, relatedBy: .equal, toItem: self.view, attribute: .width, multiplier: 1, constant: 0))
-                    self.view.addConstraint(NSLayoutConstraint(item: waitView, attribute: .width, relatedBy: .equal, toItem: self.view, attribute: .height, multiplier: 1, constant: 0))
-
-                case .landscapeLeft:
-                    self.view.addConstraint(NSLayoutConstraint(item: waitView, attribute: .height, relatedBy: .equal, toItem: self.view, attribute: .width, multiplier: 1, constant: 0))
-                    self.view.addConstraint(NSLayoutConstraint(item: waitView, attribute: .width, relatedBy: .equal, toItem: self.view, attribute: .height, multiplier: 1, constant: 0))
-
-                default:
-                    self.view.addConstraint(NSLayoutConstraint(item: waitView, attribute: .height, relatedBy: .equal, toItem: self.view, attribute: .height, multiplier: 1, constant: 0))
-                    self.view.addConstraint(NSLayoutConstraint(item: waitView, attribute: .width, relatedBy: .equal, toItem: self.view, attribute: .width, multiplier: 1, constant: 0))
-                }
-                waitView.rotateToCurrentDeviceOrientation()
-            }
-
```

Comment:
This whole block were never executed, since `imagePickerWaitingForImageDataView` is never set or used elsewhere. We figured we might as well remove it to make the file a bit more readable 🙂